stable/drupal/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: drupal
-version: 0.4.6
+version: 0.4.7
```

Comment:
@linki you make a good point. The chart version should be bumped to signal incompatibility arising from the mariadb update. Ideally I would have liked to avoid it, but... :disappointed: